apps/web/src/components/Modal/CheckOtherAddressModal.tsx

```diff
@@ -0,0 +1,19 @@
+import Modal, { ModalProps } from "@/components/Modal";
+import { LocalStorage } from "@/constants/storage";
+import useLocalStorage from "@/hooks/useLocalStorage";
+import AddressMevLookup from "../AddressMevLookup";
+
+export default function CheckOtherAddressModal(props: ModalProps) {
+  const [lookupAddress] = useLocalStorage<string | null>(LocalStorage.LOOKUP_ADDRESS, null);
+  return (
+    <Modal title="Check other address" {...props}>
+      <div className="flex flex-col gap-8 lg:w-[720px]">
```

Comment:
Can we use a tailwind variable for 720px? My suggestion would be `lg:max-w-2xl` or `lg:max-w-3xl` in conjunction with `w-full` https://github.com/tailwindlabs/tailwindcss/blob/4f9f603e12b51cc53b8a09c7739b8f88c8eb87eb/stubs/config.full.js#L653